docs/20_smart-contracts/40_guides/10_resources_guide.md

```diff
@@ -0,0 +1,232 @@
+---
+title: Resources Guide
+---
+
+The EOS blockchain works with three system resources: CPU, NET and RAM. The EOS accounts need sufficient system resources to interact with the smart contracts deployed on the blockchain.
+
+* [RAM Resource](#ram-resource)
+* [CPU Resource](#cpu-resource)
+* [NET Resource](#net-resource)
+
+To allocate resources to an account use the `PowerUp Model` to [power up that account](#account-power-up).
+
+## RAM Resource
+
+RAM is the memory, the storage space, where the blockchain stores data. If your contract needs to store data on the blockchain, like in a database, then it can store it in the blockchain's RAM in the form of a `multi-index table` or a `singleton`.
+
+### RAM High Performance
+
+The EOS blockchain high performance is achieved also because the data stored on the blockchain uses RAM as the storage medium, and the access to blockchain data is very fast. The EOS blockchain performance benchmarks reach levels very few blockchains have been able to, and it elevates EOS blockchain one of the few contenders for the fastest blockchain of the world.
+
+### RAM Importance
+
+RAM is a very important system resource because of the following reasons:
+
+* RAM is a limited resource, the public EOS blockchain started with 64GB of RAM and after that the block producers decided to increase the memory with 1KB per block. This way the supply of RAM constantly increases and its price does not grow too fast too high due to the increased demand from blockchain applications.
+
+* RAM is used in the execution of many actions sent to the blockchain. When a new account is created, for example, it needs to store in the blockchain memory the new account's information. Also when an account accepts a new type of token, a new record has to be created, somewhere in the blockchain memory, that holds the balance of the new token accepted. The blockchain memory, has to be purchased either by the account that transfers the tokens or by the account that accepts the new token type.
+
+* The smart contract can not store any additional information if it consumes all its allocated RAM. To continue to save data in the blockchain database, one, or both of the following conditions must be met:
+
+  * A portion of the occupied RAM is freed by the smart contract.
+  * More RAM is allocated to the smart contract account through the RAM purchase process.
+
+### How To Purchase RAM
+
+The RAM resource must be bought with the `EOS` system token. The price of RAM is calculated according to the unique Bancor liquidity algorithm which is implemented in the system contract [here](https://docs.eosnetwork.com/system-contracts/latest/reference/Classes/structeosiosystem_1_1exchange__state).
```

Comment:
This document linked doesn't really give much context, and I'd prefer if we didn't link to product guides from within journey guides as much as possible. The below calculation is also sufficient for what the smart contract developer will need, which is really just to get enough RAM for their purposes.